External/RingFamilies/CMakeLists.txt

```diff
@@ -23,6 +23,15 @@ set(urflib_INCLUDE_DIRS ${URFLIB_DIR}
     CACHE STRING "RingDecomposerLib Include Dir" FORCE)
 file(GLOB URFSOURCES "${URFLIB_DIR}/*.c")
 
+if((MSVC AND RDK_INSTALL_DLLS_MSVC) OR ((NOT MSVC) AND WIN32))
+    add_definitions("-DRDKIT_URFLIB_BUILD")
+    set (ringdecomposerlib_h ${URFLIB_DIR}/RingDecomposerLib.h)
+    file(READ ${ringdecomposerlib_h} ringdecomposerlib_h_data)
+    if (NOT "${ringdecomposerlib_h_data}" MATCHES "RDKIT_URFLIB_BUILD")
```

Comment:
Is it worth thinking about doing a PR against the ringdecomposerlib upstream repo so that this isn't necessary?